packages/amplify_auth_plugin_interface/lib/src/Password/ConfirmPasswordRequest.dart

```diff
@@ -13,27 +13,27 @@
  * permissions and limitations under the License.
  */
 
-import './PasswordOptions.dart';
+import 'ConfirmResetPasswordOptions.dart';
+import 'ConfirmResetPasswordRequest.dart';
 
-class ConfirmPasswordRequest {
+/// Encapsulates parameters for a confirm reset password request
+@Deprecated("Please use 'ConfirmResetPasswordRequest' instead")
+class ConfirmPasswordRequest extends ConfirmResetPasswordRequest {
   String username;
   String newPassword;
   String confirmationCode;
-  PasswordOptions? options;
+  ConfirmResetPasswordOptions? options;
```

Comment:
I don't think that `PasswordOptions` previously implemented anything, so this is probably ok? However, if a customer had this value in their code, this change would break them right?